.circleci/config.yml

```diff
@@ -1,24 +1,5 @@
 version: 2.1
 jobs:
-  test:
-    # This is a parameterized job
-    # https://circleci.com/docs/2.0/reusing-config/#authoring-parameterized-jobs
-    parameters:
-      directory:
-        type: string
-      go-image-tag:
-        type: string
-        default: "1.21-jammy"
-    description: "test at << parameters.directory >>"
-    docker:
-      - image: quay.io/cybozu/golang:<<parameters.go-image-tag>>
-      - image: quay.io/cybozu/etcd:3.5.7.1
-    working_directory: /work
-    steps:
-      - checkout
-      - run: apt-get update
-      - run: make -C << parameters.directory >> check-generate SUDO=
-      - run: make -C << parameters.directory >> test SUDO=
```

Comment:
It seems that deleting these lines does not match the purpose of this PR. Shouldn't it be deleted in a different PR?